ports/libspng/vcpkg.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "libspng",
+  "version-string": "0.6.3",
+  "description": "libspng (simple png) is a C library for reading and writing Portable Network Graphics (PNG) format files with a focus on security and ease of use."
```

```suggestion
  "description": "libspng (simple png) is a C library for reading and writing Portable Network Graphics (PNG) format files with a focus on security and ease of use.",
  "dependencies": [
    "zlib"
  ]
```
The windown failure log says: `  Could NOT find ZLIB (missing: ZLIB_LIBRARY ZLIB_INCLUDE_DIR)`